src/TrustedShops.php

```diff
@@ -156,9 +156,9 @@ public static function renderBadge() {
   /**
    * Adds trusted-shop-badge to the thank-you page.
    *
-   * @implements woocommerce_order_items_table
+   * @implements woocommerce_order_details_after_order_table_items
    */
-  public static function woocommerce_order_items_table() {
+  public static function addsTrustedShopsBuyerProtection() {
```

Comment:
The method name simply could have been renamed to its new name `woocommerce_order_details_after_order_table_items`?